news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md

```diff
@@ -0,0 +1,153 @@
+
+
+### Back office
+
+* [#13144](https://github.com/PrestaShop/PrestaShop/pull/13144): Cms page fixes. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#13251](https://github.com/PrestaShop/PrestaShop/pull/13251): Removes unnecessary return on update everywhere. Thank you [@tomas862](https://github.com/tomas862)
+* [#13264](https://github.com/PrestaShop/PrestaShop/pull/13264): Restores deleted generatable_text_widget from template. Thank you [@sarjon](https://github.com/sarjon)
+* [#13273](https://github.com/PrestaShop/PrestaShop/pull/13273): Fix domain errors, by [@matks](https://github.com/matks)
+* [#13349](https://github.com/PrestaShop/PrestaShop/pull/13349): Make all grids responsive, by [@eternoendless](https://github.com/eternoendless)
+* [#13363](https://github.com/PrestaShop/PrestaShop/pull/13363): Fix syntax error in tpl, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Tests
+
+* [#13356](https://github.com/PrestaShop/PrestaShop/pull/13356): Adds unit tests for FormHandlerFactory. Thank you [@sarjon](https://github.com/sarjon)
+
+
+### Install
+
+* [#13223](https://github.com/PrestaShop/PrestaShop/pull/13223): Fix SQL syntax error in upgrade file to PS 1.7.6, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+## Code changes in the 'develop' branch
+
+### Core
+
+* [#13119](https://github.com/PrestaShop/PrestaShop/pull/13119): Allow hookAdditionalCustomerFormFields and --AddressFormFields to modify fields positions. Thank you [@rblaurin](https://github.com/rblaurin)
+* [#13200](https://github.com/PrestaShop/PrestaShop/pull/13200): Add SecurityAnnotation linter. Thank you [@sarjon](https://github.com/sarjon)
+* [#13222](https://github.com/PrestaShop/PrestaShop/pull/13222): Do not try to install an unsupported language, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#13232](https://github.com/PrestaShop/PrestaShop/pull/13232): [CO] Wrong calculation, free Gift. Thank you [@123monsite-regis](https://github.com/123monsite-regis)
+* [#13235](https://github.com/PrestaShop/PrestaShop/pull/13235): Update src folder license headers, by [@matks](https://github.com/matks)
+* [#13237](https://github.com/PrestaShop/PrestaShop/pull/13237): Update tests folders license headers, by [@matks](https://github.com/matks)
+* [#13238](https://github.com/PrestaShop/PrestaShop/pull/13238): Update themes folders license headers, by [@matks](https://github.com/matks)
+* [#13239](https://github.com/PrestaShop/PrestaShop/pull/13239): Update js folder license headers, by [@matks](https://github.com/matks)
+* [#13242](https://github.com/PrestaShop/PrestaShop/pull/13242): Add missing hook description for additionalCustomerAddressFields, by [@eternoendless](https://github.com/eternoendless)
+* [#13359](https://github.com/PrestaShop/PrestaShop/pull/13359): Merge 1.7.6.x to develop - 11/04/2019, by [@matks](https://github.com/matks)
```

Comment:
I think we can remove it as this is a "transfer PR" whose only goal is to retrieve commits from 1.7.6.x to develop